modules/ROOT/pages/dw-mime-types.adoc

```diff
@@ -0,0 +1,35 @@
+= Mime Types (dw::module::Mime)
+
```

Comment:
```suggestion

_Introduced in DataWeave version 2.7.0._

```

Reply:
I would keep this per function. Because if we introduce new ones we will want to say 2.8 or whatever.